tests/python/pytorch/graphbolt/impl/test_fused_csc_sampling_graph.py

```diff
@@ -1848,7 +1852,7 @@ def test_sample_neighbors_replace(
 
 @unittest.skipIf(
     F._default_context_str == "gpu",
-    reason="Graph is CPU only at present.",
+    reason="Heterogenous sampling on gpu is not supported yet.",
 )
 @pytest.mark.parametrize("labor", [False, True])
```

Comment:
Does this test not work for the GPU? What is the error? If you tell me, maybe I can fix it in a quick PR.

Reply:
Just tested it, it seems to work so we can enable this test for GPU as well.